docs_user/modules/proc_adopting-autoscaling.adoc

```diff
@@ -72,10 +53,27 @@ openstack alarm create --name high_cpu_alarm \
 --threshold 7000000000
 ----
 
-* Verify the state of alarm
+.Verification
+
+. If autoscaling services are enabled, inspect the aodh pods:
++
+----
+$ AODH_POD=`oc get pods -l service=aodh -n openstack | tail -n 1 | cut -f 1 -d' '`
+$ oc exec -t $AODH_POD -c aodh-api -- cat /etc/aodh/aodh.conf
+----
+
+. Check whether the aodh API service is registered in the {identity_service}:
++
+----
+$ openstack endpoint list | grep aodh
+| d05d120153cd4f9b8310ac396b572926 | regionOne | aodh  | alarming  | True    | internal  | http://aodh-internal.openstack.svc:8042  |
+| d6daee0183494d7a9a5faee681c79046 | regionOne | aodh  | alarming  | True    | public    | http://aodh-public.openstack.svc:8042    |
+----
+
+. Verify that the alarm is enabled:
 +
```

Comment:
Does it make sense to put the "alarm verification" step as the last step in the Verification section?